documentation/Installation.md

```diff
@@ -4202,7 +4202,7 @@ The plugin configuration supports the following parameters:
 |------------------------|---------|-------------------------------------|--------------------------------------------------|--------------------------------------------------------------------|
 | mode                   | string  | `ipip`                              | `ipip` / `vxlan` / `none`                        | Network overlay technology to be used in network plugin                      |
 | crossSubnet            | boolean | `true`                              | true/false                                       | Enables crossing subnet boundaries to avoid additional network configuration. The parameter works when `mode` is set to `ipip` or `vxlan`. See more details in [Overlay networking](https://docs.tigera.io/calico/latest/networking/configuring/vxlan-ipip)|
-| mtu                    | int     | `1440`                              | MTU size on interface - 50                       | MTU size for Calico interface                                      |
+| mtu                    | int     | `1430`                              | MTU size on interface - 50                       | MTU size for Calico interface                                      |
```

Comment:
> Calico IPIP MTU size should be equal to 1430 then the MTU size if network interface is 1450
in this case 'MTU size on interface - 50 ' is not correct statement. Pls check and provide complete explanation